src/scheduler/TimeGrid.js

```diff
@@ -319,21 +319,24 @@ export default class TimeGrid extends Component {
         content.scrollLeft =
           (content.scrollWidth - gutterWidth) * this._scrollRatio
         // Only do this once
-        this._scrollRatio = null
```

Comment:
I'd assume that removing this may keep the calendar "locked" at a certain position as it will always attempt to scroll but that doesn't seem to be the case.